diagrams/create_request_uri.md

```diff
@@ -0,0 +1,50 @@
+```plantuml
+@startuml
+
+autonumber
+
+box "Wallet"
+participant "Metadata" as wm
+participant "Authorization Endpoint" as w
+end box
+
+participant "User Agent" as u
+
+box "Verifier"
+participant "Frontend" as r
+participant "Request Endpoint" as rp
+participant "Response Endpoint" as rb
+end box
+
+u --> r : use
+activate r
+
+r --> u: **signed authorization request**\n(client_id, create_request_uri, state)
+deactivate r
+u --> w: **signed authorization request**\n(client_id, create_request_uri, state)
+activate w
+w --> w: check authorization request signature
+w --> w: check on trustworthiness of Verifier (approach on trust mechanism)
+w --> rp: POST **create request request** (\nstate,\n[OPTIONAL]issuer, \n[OPTIONAL]wallet_metadata, \n[OPTIONAL]w_nonce, \n[OPTIONAL]w_ephm_key, \n[OPTIONAL]wallet attestation, wallet attestation pop(v_nonce))
+note over u, w: HTTP status code 401 signals need to authenticate
+rp --> wm: [OPTIONAL] get wallet metadata
+wm --> rp: [OPTIONAL] wallet metadata
+rp -> rp: create and sign presentation request object (client_id, w_nonce, nonce, response_uri, \npresentation_definition, state)
+rp --> w: **signed request object** (client_id, w_nonce, nonce, response_uri, \npresentation_definition, state)
+note over u, w: do we want to allow unsigned presentation request objects, too?
```

Comment:
Since we had a signed authz request and presumably a secure channel then signing the presentation request object should be optional

Reply:
you are right, we already have authenticated the verifier at that stage. 

What do others think? Should we make the signature of the presentation request optional?

Reply:
I think in the meeting last week, we agreed that having unsigned requests is a viable approach but we should create a new issue for that, after we merged this PR.